services/addons/images/obu_ota_server/obu_ota_server.py

```diff
@@ -1,23 +1,23 @@
+from typing import Any

```

Comment:
Does this library need to be added to requirements.txt?

Reply:
It shouldn't have to be, the typing library is included with python back to python 3.5